docs/community/jobcenter.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How to Post A Job
+sidebar_position: 7
+---
+
+# Job Center
+
+The KubeEdge [Job Center](https://kubeedge.io/job-center/) aims to provide a platform for community partners and developers to exchange job opportunities. Partners can post job openings here and provide detailed job information and requirements, while community users can select job positions based on their experience and interests. We sincerely hope that you can find suitable jobs or excellent talents here.
+
+This guide is intended to help you understand how to post your job positions in the community job center and provide guidelines for writing job information documents.
+
+# Requirement for Posting Jobs
+
+- You must be a contributing organization or adopter of the KubeEdge community, including but not limited to having used KubeEdge in your company's public products or public solutions, contributing to the community, and submitting your use cases.
+- Your company/organization logo has been added to the [KubeEdge Supporters](https://kubeedge.io/#supporters).
+
+# Writing Guidelines
+
+We hope that you can provide as much detailed information as possible about the job.
+
+1. Please add the following information at the top of the document. This information will be displayed in a card format on the KubeEdge official website's job center homepage, which will help community users quickly understand the key information about the job.
```

Comment:
same here, https://github.com/kubeedge/community/pull/148/files#r1370852333? i think it would be better for end user experience, where to change in the 1st place.

Reply:
> same here, https://github.com/kubeedge/community/pull/148/files#r1370852333? i think it would be better for end user experience, where to change in the 1st place.



> same here, https://github.com/kubeedge/community/pull/148/files#r1370852333? i think it would be better for end user experience, where to change in the 1st place.

Thanks for your suggestions. The previous introduction here is really confusing, and I've fixed it.